find-route/src/main/res/layout/activity_main.xml

```diff
@@ -38,10 +38,12 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_margin="16dp"
+                android:layout_marginEnd="30dp"
```

Comment:
Setting `layout_margin` overrides values set for layout_marginEnd and layout_marginBottom. 

I think we can remove these.